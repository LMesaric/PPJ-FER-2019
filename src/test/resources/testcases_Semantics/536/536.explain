semantic: invalid escaped string '\o' on byte offset 1
note: ''\o'' on line 19
