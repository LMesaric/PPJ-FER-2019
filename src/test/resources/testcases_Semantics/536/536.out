<primarni_izraz> ::= ZNAK(19,'\o')
